Src/CSharpier/DocPrinter/DocFitter.cs

```diff
@@ -23,7 +23,7 @@ void Push(Doc doc, PrintMode printMode, Indent indent)
 
         var output = new StringBuilder();
 
-        for (var x = 0; x < remainingCommands.Count || newCommands.Count > 0;)
+        for (var x = 0; x < remainingCommands.Count || newCommands.Count > 0; )
```

Comment:
Is this a bug?

Reply:
No, I made the change to get csharpier to abide by the stylecop rule for always having a space after a semicolon. Prettier formats for's without iterators the same way.